framework/src/controller/bus.ts

```diff
@@ -231,30 +262,24 @@ export class Bus extends EventEmitter2 {
 		}
 
 		// Communicate through event emitter
-		this.on(eventName, cb);
+		this._emitter.on(eventName, cb);
 
-		// Communicate through unix socket
-		if (this.config.ipc.enabled) {
-			this.subSocket?.on(eventName, cb);
-		}
+		// // Communicate through unix socket
+		// if (this.config.ipc.enabled) {
```

Comment:
Originally there was option to allow communication directly through socket or not, but that's not the case anymore?

Reply:
Now this check is only in one place inside `setup`, rest is handled through local emitter.